src/app/shared/utils/route.utils.ts

```diff
@@ -0,0 +1,6 @@
+import { Router } from '@angular/router';
+
+export function currentPath(router: Router) {
```

Comment:
Please add TypeDocs to this function.  It's a simple function, but it'd be good to still note exactly that it is doing.

Reply:
Added TypeDoc